curp/src/client.rs

```diff
@@ -109,11 +109,12 @@ where
 
     /// The fast round of Curp protocol
     /// It broadcast the requests to all the curp servers.
-    #[instrument(skip(self))]
+    #[instrument(skip_all)]
```

Comment:
If you want to print the `cmd.id()` instead of the `cmd`, you can use the adding field. FYI: https://docs.rs/tracing/0.1.38/tracing/attr.instrument.html#adding-fields

```suggestion
    #[instrument(skip(self), field(cmd_id = cmd_arc.id()) )]
```

Reply:
Fixed